.travis.yml

```diff
@@ -4,9 +4,9 @@ os:
   - osx
 sudo: false
 go:
+  - "1.12.x"
   - "1.11.x"
-  - "1.10.x"
-env: CGO_ENABLED=0
+env: CGO_ENABLED=0 GO111MODULE=on
```

Comment:
Before `GO111MODULE` defaults to on, we should support both GOPATH-based and module-based workflows. So we will need to test two cases, one where GO111MODULE is not set (and because of the setup in our travis script, the GOPATH workflow will be used) and where it is set to be `on`.

Reply:
I've changed it here, as this won't work properly with Go 1.10. It's an "all or nothing" with regards to converting over to modules.